lib/src/components/NxDonutChart/NxDonutChart.tsx

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React, { FunctionComponent, ReactElement } from 'react';
+import { Props, propTypes } from './types';
+export { Props } from './types';
+
+const NxDonutChart: FunctionComponent<Props> =
```

Comment:
Given that we expect to have more full-fledged donut charts in the future (ones with more than two slices), I wonder if this should have a slightly more specific name, like `NxCompletionDonutChart` or `NxProgressDonutChart`

Reply:
> Given that we expect to have more full-fledged donut charts in the future (ones with more than two slices), I wonder if this should have a slightly more specific name, like `NxCompletionDonutChart` or `NxProgressDonutChart`

I thought `NxBinaryDonutChart` might also be alright if that works for you https://github.com/sonatype/sonatype-react-shared-components/pull/172/commits/47355af1906408054b5f20f7a7b2cea5319f34f5